src/scenes/Event/components/TransactionHistory/TxRow.js

```diff
@@ -72,23 +69,28 @@ export default class TxRow extends Component {
 
     return (
       <Fragment>
-        <TableRow key={`tx-${txid}`} onClick={this.toggle}>
+        <TableRow key={`tx-${txid}`}>
```

Comment:
don't we support click the row to expand?

Reply:
I moved the click to the arrow icon only because it felt like bad UI when I wanted to copy and address or some other text, it would keep toggling the expansion.